src/RestApi/StoreApi/Controllers/Cart.php

```diff
@@ -103,8 +103,57 @@ public function register_routes() {
 				'schema' => [ $this, 'get_public_item_schema' ],
 			]
 		);
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/select-shipping',
```

Comment:
Minor point, since the other endpoint is `/cart/shipping-rates` maybe this should be `cart/select-shipping-rates`. Do you think that makes sense?